app/controllers/application_controller.rb

```diff
@@ -2,4 +2,5 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  include SessionsHelper
```

Comment:
generally bad form to add anything to application controller.

what is sessionshelper? is that a rails thing?